smartMeter/jsonld-contexts/smartMeter.jsonld

```diff
@@ -0,0 +1,45 @@
+{
+    "@context": {
+        "Consumer": "https://uri.fiware.org/ns/data-models#Consumer",
```

Comment:
shouldn't all contexts be based on smart-data-models domain?

Reply:
changed temporarily to https://ontology.eglobalmark.com/smartMeter#